packages/react/src/components/Breadcrumb/Breadcrumb.Skeleton.js

```diff
@@ -8,24 +8,27 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
-import { settings } from 'carbon-components';
+import { usePrefix } from '../../internal/usePrefix';
 
-const { prefix } = settings;
+const item = () => {
+  const prefix = usePrefix();
 
-const item = (
-  <div className={`${prefix}--breadcrumb-item`}>
-    <span className={`${prefix}--link`}>&nbsp;</span>
-  </div>
-);
+  return (
+    <div className={`${prefix}--breadcrumb-item`}>
+      <span className={`${prefix}--link`}>&nbsp;</span>
+    </div>
+  );
+};
 
 function BreadcrumbSkeleton({ className, ...rest }) {
+  const prefix = usePrefix();
   const classes = cx(`${prefix}--breadcrumb`, `${prefix}--skeleton`, className);
 
   return (
     <div className={classes} {...rest}>
-      {item}
-      {item}
-      {item}
+      {item()}
+      {item()}
+      {item()}
```

Comment:
These should be changed to use JSX syntax after `item` is changed to a component